includes/class-amp-story-post-type.php

```diff
@@ -1036,14 +1036,13 @@ public static function the_single_story_card( $args ) {
 		?>
 		<<?php echo esc_attr( $wrapper_tag_name ); ?> <?php echo isset( $href ) ? wp_kses_post( $href ) : ''; ?> class="latest_stories__link">
 			<?php
-			echo wp_get_attachment_image(
-				$thumbnail_id,
-				$args['size'],
-				false,
-				array(
-					'alt'   => get_the_title( $post ),
-					'class' => 'latest-stories__featured-img',
-				)
+			list( $src ) = wp_get_attachment_image_src( $thumbnail_id, $args['size'] );
```

Comment:
Nice use of `list()`

Reply:
Well, I updated the code to just use `wp_get_attachment_image_url()` instead :-)

Reply:
Good point. I guess I'm too old school.